src/index.tsx

```diff
@@ -32,7 +32,7 @@ function configureMonacoEnvironment() {
   };
 }
 
-async function bootstrap() {
+(async () => {
```

Comment:
Why did you change `bootstrap` function to something like that?

Reply:
SonarCloud complains (marks as a bug) with something like this iirc: "Promises must be awaited, end with a call to .catch, or end with a call to .then with a rejection handler"

Reply:
As we see, Sonarcloud was written by Java people 😆 Ok.